bears/configfiles/DockerfileLintBear.py

```diff
@@ -9,6 +9,12 @@
 class DockerfileLintBear:
     """
     Checks the given file with ``dockerfile_lint``.
+
+    Can be used to check file syntax as well as arbitrary semantic and best
+    practice attributes. The linter can also be used to check LABEL rules
```

Comment:
I copied it from the dockerfile_lint github page, I think it's some programming term so I would keep it^^